tests/dbz-twin/rac/db-check.py

```
@@ -0,0 +1,383 @@
+
+
+def get_event_count(conn, source_table):
+    """Get total event count."""
+    return conn.execute(f"SELECT COUNT(*) FROM {source_table}").fetchone()[0]
+
+
+def refresh_db(sqlite_path):
+    """Re-copy SQLite DB from consumer container (including WAL)."""
+    import subprocess
+    d = os.path.dirname(sqlite_path)
+    subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)